Source/Scene/Camera.js

```diff
@@ -2738,6 +2738,7 @@ define([
     Camera.clone = function(camera, result) {
         if (!defined(result)) {
             result = new Camera(camera._scene);
+            result.frustum = camera.frustum.clone();
```

Comment:
I'm not totally sure. I don't know enough about the 3D to 2D transition yet, but I was thinking shadows would always operate with 3D values, in which case there would have to be a perspective camera lying around.